src/server/templates/result-event-evaluation.html

```diff
@@ -1,105 +1,344 @@
+        class="detail-button less-visible7"
+        style="display: none;"
+      >
+        Close Calculation <i class="fas fa-caret-up"></i>
+      </button>
+    </div>
+  </div>
+  <div
+    id="calculationProposedMonth"
+    class="row less-visible8"
+    style="display: none;"
+  >
+    <div class="col-12">
+      {{proposedMonthCalculation | default("[proposedMonthCalculation]")}}
+    </div>
+  </div>
+  <br />
+  <div class="row">
+    <div class="col-12">
+      {{roposedMonthGraph | default("[roposedMonthGraph]")}}
+    </div>
+  </div>
+  <br />
 
-  <!-- graph -->
+  <div class="row">
+    <div class="col-12">
+      <h1 class="title-form">Did this change your mind at all?</h1>
+    </div>
+  </div>
 
-  <label>Did this change your mind at all?</label>
   <!--form-->
-  <form action="/decision">
-    <div class="form-group">
-      <div class="btn-group-toggle" data-toggle="buttons">
-        <label class="btn btn-secondary">
-          <input type="radio" name="more" id="more" autocomplete="off" />
-          Yes, more likely to host
-        </label>
-        <label class="btn btn-secondary">
-          <input type="radio" name="less" id="less" autocomplete="off" />Yes,
-          less likely to host
-        </label>
-        <label class="btn btn-secondary">
-          <input type="radio" name="equal" id="equal" autocomplete="off" />No
-        </label>
-      </div>
+  <div class="row">
+    <div class="col-12">
+      <form action="/decision">
+        <div class="form-group">
+          <div class="row btn-group-toggle" data-toggle="buttons">
+            <div class="col-md-auto col-sm-12">
+              <label class="btn btn-secondary submit">
+                <input type="radio" name="more" id="more" autocomplete="off" />
+                Yes, more likely to host
+              </label>
+            </div>
+            <div class="col-md-auto col-sm-12">
+              <label class="btn btn-secondary submit">
+                <input
+                  type="radio"
+                  name="less"
+                  id="less"
+                  autocomplete="off"
+                />Yes, less likely to host
+              </label>
+            </div>
+            <div class="col-md-auto col-sm-12">
+              <label class="btn btn-secondary submit">
+                <input
+                  type="radio"
+                  name="equal"
+                  id="equal"
+                  autocomplete="off"
+                />No
+              </label>
+            </div>
+          </div>
+        </div>
+      </form>
     </div>
-  </form>
-  <p class="form-submition-message"></p>
+  </div>
+  <div class="row" style="margin-top: -16px;">
+    <div class="col-12">
+      <!-- Message confirming the answer has been submitted -->
+      {{formSubmitionMessage}}
+    </div>
+  </div>
 
-  <div id="credits">
-    <h2>Credits</h2>
-    <p>
+  <br />
```

Comment:
again, instead of `br`s I would use `mt` or `mb` (https://getbootstrap.com/docs/4.4/utilities/spacing/)